Patch-Source: https://github.com/django-q2/django-q2/pull/131
From 2c2594348ebed7194afdda31e1d303f24f68935c Mon Sep 17 00:00:00 2001
From: ptrcnull <git@ptrcnull.me>
Date: Thu, 19 Oct 2023 17:06:29 +0200
Subject: [PATCH] Specify build system in pyproject.toml

Otherwise, trying to build the project with a tool like gpep517 or build
results in using the setuptools legacy backend instead

diff --git a/pyproject.toml b/pyproject.toml
index e785688..8de3163 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,3 +1,7 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
 [tool.poetry]
 name = "django-q2"
 version = "1.5.4"
